static/js/rendered_markdown.js

```diff
@@ -137,11 +137,11 @@ exports.update_elements = (content) => {
         }
     });
 
-    content.find('span.timestamp').each(function () {
+    content.find('.timestamp').each(function () {
         // Populate each timestamp span with mentioned time
         // in user's local timezone.
-        const timestamp = moment.unix($(this).attr('data-timestamp'));
-        if (timestamp.isValid() && $(this).attr('data-timestamp') !== null) {
+        const timestamp = moment($(this).attr('datetime'));
+        if (timestamp.isValid() && $(this).attr('datetime') !== undefined) {
```

Comment:
This was present before, but it doesn't make sense to check if something is undefined AFTER we've passed it to `moment`.  We should just check for the attribute and continue if not present before calling `moment` on it.

Reply:
This was done to avoid having nested ifs, I think. I recall we discussed this briefly and then decided to go this route, with calculating the moment object before the if.